cmd/cluster-image-registry-operator/main.go

```diff
@@ -37,6 +42,15 @@ func watch(apiVersion, kind, namespace string, resyncPeriod time.Duration) {
 func main() {
 	flag.Parse()
 
+	if logLevel == nil {
+		*logLevel = os.Getenv("LOG_LEVEL")
+	}
+	lvl, err := logrus.ParseLevel(*logLevel)
```

Comment:
I would expect command line arguments to have precedence over environment variables. Though, the default value for the command line argument shouldn't be taken into account.